pyccel/codegen/printing/pycode.py

```diff
@@ -215,59 +219,47 @@ def _print_PythonList(self, expr):
         args = ', '.join(self._print(i) for i in expr.args)
         return '['+args+']'
 
-    def _print_PythonLen(self, expr):
-        return 'len({})'.format(self._print(expr.arg))
-
-    def _print_PythonAbs(self, expr):
-        return 'abs({})'.format(self._print(expr.arg))
-
-    def _print_PythonMin(self, expr):
-        return 'min({})'.format(self._print(expr.args[0]))
-
-    def _print_PythonMax(self, expr):
-        return 'max({})'.format(self._print(expr.args[0]))
-
-    def _print_PythonSum(self, expr):
-        return 'sum({})'.format(self._print(expr.args[0]))
-
     def _print_PythonBool(self, expr):
         return 'bool({})'.format(self._print(expr.arg))
 
     def _print_PythonInt(self, expr):
         type_name = type(expr).__name__.lower()
         is_numpy  = type_name.startswith('numpy')
         precision = str(expr.precision*8) if is_numpy else ''
-        name = 'int{}'.format(precision)
+        name = self._aliases.get(expr.name, expr.name)
         if is_numpy:
-            self.insert_new_import(Import('numpy',name))
+            self.insert_new_import('numpy',expr.name,name)
```

Comment:
I think that this function call would be much easier to understand if you passed the arguments with keywords.

Reply:
Done 480851c

Reply:
Modified e5ec188